rescript/tests/test_get_eukaryome.py

```diff
@@ -0,0 +1,79 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2019-2025, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+import pkg_resources
+from qiime2.plugin.testing import TestPluginBase
+from qiime2.plugins import rescript
+from rescript.get_eukaryome import (_assemble_rrna_url,
+                                    _retrieve_data_from_eukaryome)
+from q2_types.feature_data import (TSVTaxonomyFormat,
+                                   DNAFASTAFormat)
+from unittest.mock import patch
+
+
+class TestGetEukaryome(TestPluginBase):
+    package = 'rescript.tests'
+
+    def setUp(self):
+        super().setUp()
+        self.eukaryome_seqs = DNAFASTAFormat(
+                            pkg_resources.resource_filename(
+                                 'rescript.tests',
+                                 'data/eukaryome-seqs.fasta'),
```

Comment:
```suggestion
                            self.get_data_path('eukaryome-seqs.fasta'),
```

Please replace all the paths to test files with `self.get_data_path`. A bit more elegant.